.travis.yml

```diff
@@ -49,6 +47,12 @@ jobs:
         - sudo operator-sdk build quay.io/pulp/pulp-operator:latest
         - sudo docker images
         - sudo insta-demo/pulp-insta-demo.sh
+    - name: molecule
+      dist: xenial
+      install:
+        - pip3 install docker molecule ansible-lint yamllint flake8 openshift jmespath
+      before_script: skip
+      script: .travis/molecule.sh
```

Comment:
Shouldn't this be part of the test-all-components job? So that a molecule failure blocks it from being pushed to quay?

Ideally the push to quay should be a separate job after the others, but still.

Reply:
Good point, I forgot about the deploy!

Reply:
Since we still rely on the ports being changed, we should just run molecule as a separate test, like is currently done.

See comment about the push-to-quay stage though.